artemis-commons/src/main/java/org/apache/activemq/artemis/api/core/SimpleString.java

```diff
@@ -150,6 +150,10 @@ public CharSequence subSequence(final int start, final int end) {
       return subSeq(start, end);
    }
 
+   public static boolean compare(SimpleString s1, SimpleString s2) {
```

Comment:
you can call it equals as well, `compare` usually is used for `Comparator` API-like ie that return a `int` value

Reply:
Good point, this isn't even needed, changed to using java.util.Objects.equals